frontend/vue/components/Syllabus/SyllabusGeneralInformation.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="syllabus-general-information">
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">Instructor:</span> <slot name="instructor" />
```

Comment:
I believe we should be using props instead of slots since we're always expecting text strings. We could then profit from type checking and also easily cover the scenarios where one of the attributes isn't defined with `v-if`.

We should then also simplify this repetitive sections with a `v-for` loop.